### PR TITLE
Clarification of init-container restart reasons #13022

### PR DESCRIPTION
**Pull request by SupriyaSirbi**
Clarify restart reasons

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ same as the scheduler.
 A Pod can restart, causing re-execution of Init Containers, for the following
 reasons:
 
-* A user updates the PodSpec causing the Init Container image to change.
+* A user updates the PodSpec causing the Init Container image to restart the entire pod.
```
Comment (Rajakavitha1) [inline content/en/docs/concepts/workloads/pods/init-containers.md]: Suggestions:  A user updates the PodSpec causing the Init Container image to change. Any changes to the Init Container image restarts the Pod.

Based on:  Altering an Init Container image field is equivalent to restarting the Pod.

Comment (SupriyaSirbi) [reply content/en/docs/concepts/workloads/pods/init-containers.md]: Hello Rajakavitha,

Done. Thanks for your comments

File: content/en/docs/concepts/workloads/pods/init-containers.md [hunk 2]
```diff
@@ -318,8 +318,9 @@ same as the scheduler.
 A Pod can restart, causing re-execution of Init Containers, for the following
 reasons:
 
-* A user updates the PodSpec causing the Init Container image to restart the entire pod.
-  App Container image changes only restart the app Container.
+* A user updates the PodSpec causing the Init Container image to change. Any
+  changes to the Init Container image restarts the Pod.App Container image
```
Comment (Rajakavitha1) [inline content/en/docs/concepts/workloads/pods/init-containers.md]: @SupriyaSirbi  nit: Please add a (space) after .....Init Container image restarts the Pod.

**Comment by Rajakavitha1** [comment]
@SupriyaSirbi nit: Please add a (space) after .....Init Container image restarts the Pod.

File: content/en/docs/concepts/workloads/pods/init-containers.md (hunk 2 above)
Comment (SupriyaSirbi) [reply content/en/docs/concepts/workloads/pods/init-containers.md]: ok done

**Comment by Rajakavitha1** [comment]
Thanks for the PR @SupriyaSirbi !!!!
/lgtm
/approve

**Pull request merged**